pkg/predictor_source/inferenceservice_registry.go

```diff
@@ -68,6 +69,55 @@ func BuildBasePredictorFromInferenceService(isvc *v1beta1.InferenceService) (*v1
 	return p, nil
 }
 
+// Return secretKey, bucket, modelPath, schemaPath, and error
+func processInferenceServiceStorage(inferenceService *v1beta1.InferenceService, nname types.NamespacedName) (string, string, string, string, error) {
+	var bucket string
+	var modelPath string
+	var schemaPath string
+	secretKey := inferenceService.ObjectMeta.Annotations[v1beta1.SecretKeyAnnotation]
```

Comment:
```suggestion
func processInferenceServiceStorage(inferenceService *v1beta1.InferenceService, nname types.NamespacedName) (secretKey, bucket, modelPath, schemaPath string, err error) {
	secretKey = inferenceService.ObjectMeta.Annotations[v1beta1.SecretKeyAnnotation]
```
This also means that you can remove the parameters from the final `return`.